### ISSUE TITLE
Reference in `circular_shifts`

### ISSUE THREAD

**Issue by pylang** [erikrose/more-itertools#197]
`circular_shifts(iterable)` is a specific case of `windowed`, e.g. 

```python
list(it.islice(mit.windowed(it.cycle(iterable), n=len(iterable)), len(iterable)))

list(mit.windowed(mit.ncycles(iterable, n=2), n=len(iterable)))[:-1]
```

I suggest a reference in the `circular_shifts` docstrings to "See also `windowed`".

**Comment by bbayles** [comment]
Hmm, I see `circular_shifts` as a clever application of `cycle` and `windowed` (and your alternate here is also clever), but not really a special case?

I would like to get links to the source working on the ReadTheDocs. Somehow I got that to work for [redis-collections](https://redis-collections.readthedocs.io/en/stable/api.html), but I have no idea how. [sphinx.ext.viewcode](https://github.com/honzajavorek/redis-collections/blob/master/docs/conf.py#L28) does seem to be [in place](https://github.com/erikrose/more-itertools/blob/master/docs/conf.py#L30), but I'm not sure why it's working. PR welcome!

**Comment by bbayles** [comment]
Addressed for the curious by PR #198.
